src/test/java/org/ahocorasick/trie/PayloadTrieTest.java

```diff
@@ -449,8 +449,9 @@ public void testLargeString() {
         assertEquals(textSize / interval, emits.size());
     }
 
+    // @see https://github.com/robert-bor/aho-corasick/issues/85
     @Test
-    public void testWholeWords() {
+    public void wholeWords() {
```

Comment:
Please revise to use Roy's naming scheme: https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html, prefixed with `test_`.

Reply:
👍